client/my-sites/marketing/index.web.js

```diff
@@ -12,36 +11,32 @@ import {
 	redirectSharingButtons,
 	sharingButtons,
 	traffic,
-	ultimateTrafficGuide,
 } from './controller';
 
-export default function () {
-	page( '/marketing/do-it-for-me*', function redirectToDIFMLandingPage() {
-		window.location.replace( 'https://wordpress.com/do-it-for-me' );
-	} );
-
+export default function ( router ) {
 	const paths = [
 		'/marketing',
 		'/marketing/connections',
 		'/marketing/sharing-buttons',
 		'/marketing/tools',
 		'/marketing/traffic',
-		'/marketing/ultimate-traffic-guide',
 		'/sharing',
 		'/sharing/buttons',
 		'/marketing/business-tools',
 	];
 
-	paths.forEach( ( path ) => page( path, ...[ siteSelection, sites, makeLayout, clientRender ] ) );
+	paths.forEach( ( path ) =>
+		router( path, ...[ siteSelection, sites, makeLayout, clientRender ] )
+	);
 
-	page( '/marketing/connection/:service', redirectDefaultConnectionsDomain );
+	router( '/marketing/connection/:service', redirectDefaultConnectionsDomain );
```

Comment:
`page` and `router` seems to have the same signature - are they the same function?

Reply:
`router` is required for isomorphic routing, per the docs. I don't know how it's implemented, though.